src/main/java/reposense/parser/CsvParser.java

```diff
@@ -0,0 +1,126 @@
+package reposense.parser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import reposense.dataobject.Author;
+import reposense.dataobject.RepoConfiguration;
+import reposense.frontend.CliArguments;
+import reposense.util.Constants;
+
+/**
+ * Parses a CSV configuration file for repository information.
+ */
+public class CsvParser {
```

Comment:
Did you consider using a third-party csv parser?

Reply:
Currently, the Csv Parser is sufficient for our needs. We can look into third-party libraries if there exists use cases for it.